src/pages/donate.tsx

```diff
@@ -80,12 +91,14 @@ const Donate = () => {
   let souvenirs: string[] = []
```

Comment:
Please data this variable using `useState`, having a unnecessary mutable variable in react cause re-rendering problem

Reply:
Okay, that variable is now declared by `useState`